### PR TITLE
Speedup reloading interfaces.

### PR DESCRIPTION
**Pull request by kekek2**
If WAN-interface has no connection then restart of all services is very slow. The IP address is received by DHCP for a long time and the DNS server is restarted for a long time. Moreover, the DNS server is rebooted three times: once per initialization of each interface and once again on the general reboot of interfaces. I removed the unnecessary reset of the DNS servers on each interface. Restart all services accelerated in half.

### CODE REVIEW

File: src/etc/inc/interfaces.inc [hunk 1]
```diff
@@ -950,8 +954,8 @@ function interfaces_configure($verbose = false)
 
     interfaces_group_setup();
 
-    if ($reload) {
-        system_routing_configure($verbose);
+    if ($reload == INTERFACE_RELOAD_ALL) {
+        system_routing_configure('', $verbose);
```
Comment (fichtner) [inline src/etc/inc/interfaces.inc]: this has merge conflicts

File: src/etc/inc/interfaces.inc [hunk 2]
```diff
@@ -905,7 +909,7 @@ function interfaces_configure($verbose = false)
     $track6_list = array();
 
     /* This is needed to speedup interfaces on bootup. */
-    $reload = !file_exists('/var/run/booting');
+    $reload = file_exists('/var/run/booting') ? INTERFACE_BOOTING : INTERFACE_RELOAD_ALL;
```
Comment (fichtner) [inline src/etc/inc/interfaces.inc]: is that really necessary?

File: src/etc/rc.initial.setlanip [hunk 3]
```diff
@@ -558,7 +558,7 @@ echo "done.\n";
 system_hosts_generate(true);
 system_resolvconf_generate(true);
 interface_bring_down($interface);
-interface_configure($interface, true, false, true);
+interface_configure($interface, INTERFACE_RELOAD_SINGLE, false, true);
```
Comment (fichtner) [inline src/etc/rc.initial.setlanip]: isn't that determined by setting $interface in the first place?

File: src/etc/inc/interfaces.inc (hunk 2 above)
Comment (kekek2) [reply src/etc/inc/interfaces.inc]: https://github.com/kekek2/core/blob/c4b02532caf071a05bff96312041511906837920/src/etc/inc/interfaces.inc#L957

In this line uses variable $reload.

**Comment by fichtner** [comment]
After reviewing this and internal code the conclusion is that reconfiguring should be thorough, not fast. I've adjusted a few things on master, but this here won't go in.

**Pull request closed without merging**